experiments/dask/bench.yml

```diff
@@ -0,0 +1,104 @@
+# TODO
+#  * add support for item.repetitions
+- name: pull image
+  shell: docker pull {{ item.image_name }}
+- name: pull alpine
+  shell: docker pull alpine
+
+- name: drop caches
+  shell: echo 3 > /proc/sys/vm/drop_caches
+
+- name: run benchmark on localfs
+  shell: >
+    docker run --rm \
+      -v {{ item.folder_name }}:/mount \
+      {{ item.docker_flags }} \
+      {{ item.image_name }} {{ item.image_args }}
+  when: "{{ item.run_in_local_fs | bool }}"
+  register: docker_output
+
+- name: write results to output
+  shell: echo "{{ item.id }}, exec, local, {{docker_output.delta}}" >> results.csv
+  when: "{{ item.run_in_local_fs | bool }}"
+
+# gassyfs
+
+- name: copy folder into gassyfs
+  shell: >
+    docker run --rm \
+      --volumes-from gassyfs-master \
```

Comment:
Why do you use 'shell docker run blah'? Is it because Ansible doesn't support '--volumes-from'?